### PR TITLE
Switch to gh ci

### PR DESCRIPTION
**Pull request by ben-alkov**
Signed-off-by: Ben Alkov <ben.alkov@redhat.com>

- CLOUDBLD-2142

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

**Comment by ben-alkov** [comment]
Note that the new GH workflow files were copied verbatim from the atomic-reactor, so have been thoroughly tested

File: .github/workflows/linters.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+name: Linters
+
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (MartinBasti) [inline .github/workflows/linters.yaml:6]: I'd like to run tests also for release branches. Basically I want to run test for each PR

Comment (ben-alkov) [reply .github/workflows/linters.yaml:6]: OK. Same change for atomic-reactor?

Comment (ben-alkov) [reply .github/workflows/linters.yaml:6]: tldr; add 'release_*' to `on: pull_request` for linters and unittests

Comment (ben-alkov) [reply .github/workflows/linters.yaml:6]: Added

**Comment by ben-alkov** [comment]
If you need more insight into the 'test.sh' changes, I'd recommend looking at [the MR *these* changes are based on][].

[the MR *these* changes are based on]: https://github.com/containerbuildsystem/atomic-reactor/pull/1536/commits

**Review by MartinBasti — APPROVED** [review]

**Review by rcerven — APPROVED** [review]

File: pytest.ini [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[pytest]
+# addopts for html have to go in test.sh because of packit (rpm specfile runs unit tests)
```
Comment (StykMartin) [inline pytest.ini:2]: @ben-alkov you should correct this comment.
Packit is just tooling that makes build for you, however, the problem here is that you have %check running pytest in specfile. So you could get the same problems when you will use mock to create a build.

**Pull request merged**